app/src/main/java/com/example/dispatcher/presentation/HeaderView/HeaderView.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.dispatcher.presentation.HeaderView
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.RelativeLayout
+import com.example.dispatcher.R
+import com.example.dispatcher.databinding.ViewHeaderBinding
+
+class HeaderView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : RelativeLayout(context, attrs, defStyleAttr) {
+
+    private val binding: ViewHeaderBinding = ViewHeaderBinding.inflate(LayoutInflater.from(context), this, true)
```

Comment:
Please notice that the view is not like in figma. fix it to be aligned with figma.